services/app-api/emailer/templates.ts

```diff
@@ -149,5 +149,33 @@ const newPackageStateEmail = (
     }
 }
 
-export { newPackageCMSEmail, newPackageStateEmail }
+const unlockPackageCMSEmail = (
+    submission: StateSubmissionType,
```

Comment:
For this parameter instead of `submission: StateSubmissionType` I think it will be `revision: RevisionType`. You can go ahead and add that and you can find that type (so far key, fields are missing that @JasonLin0991  is adding) in domain-models/Submission2Type

@JasonLin0991 could you tell us what the `unlockInfo` and `submitInfo` will look like after your changes. That could unblock josh to keep working here.


If we find out more about what the revisions type will look like then the next step with this PR could be writing tests.